robot/Cumulus/tests/browser/gift_entry/checkbox_picklists.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/ContactPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             API Check And Enable Gift Entry
+Suite Teardown  Run Keywords
+...             Delete Session Records
+...             Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+
+
+*** Keywords ***
+Setup Test Data
+    ${NS} =  Get NPSP Namespace Prefix
+    Set suite variable    ${NS}
+
+    &{CONTACT}=  API Create Contact
+    ...          FirstName=${faker.first_name()}
+    ...          LastName=${faker.last_name()}
+    ...          Checkbox_to_picklist__c=true
+    Store Session Record     Contact     ${CONTACT}[Id]
+    Set Suite Variable  &{CONTACT}
+
+*** Test Cases ***
+
+Verify Checkbox to Checkbox Field Mappings Are Successful
+  [Documentation]
+  [Tags]                                W-8789974    unstable        feature:GE      
+  ${template} =                         Generate Random String
+  Go to Page                            Landing     GE_Gift_Entry
+  Click Link                            Templates
+  Click Gift Entry Button               Create Template
+  Current Page Should Be                Template                    GE_Gift_Entry
+  Enter Value in Field
+  ...                                   Template Name=${template}
+  ...                                   Description=This is created by automation script 
+  Click Gift Entry Button               Next: Form Fields
+  Perform Action on Object Field        select   Contact 1  Contact1 Checkbox To Checkbox
+  Perform Action on Object Field        select   Contact 1  Contact1 Checkbox To Picklist
+  Page Should Not Contain Element       //lightning-input[@data-qa-locator="checkbox Required Contact 1: Contact1 Checkbox To Checkbox"]
```

Comment:
please move this locator to locators file and use locator manager strategy in the doc. We also have some Gift Entry tests thats using this strategy already https://salesforce.quip.com/oyjJAcFaWtm8

Reply:
Done!